Improve export troubleshooting page (#10638)
braze-inc/braze-docs: _docs/_user_guide/data/distribution/export_braze_data/export_troubleshooting.md

article_title: Export Troubleshooting
page_order: 10
page_type: reference
description: "This reference article covers common troubleshooting scenarios for exports in both CSV and API workflows."
---

# Export troubleshooting

> This page covers common troubleshooting scenarios for exports in both CSV and API workflows.  

Use the tabs to select whether you're exporting to the **default Braze S3 bucket** or to a **cloud storage partner**.

{% sdktabs %}
{% sdktab Default export %}

When you don't have a storage partner marked as your default export destination, Braze uses its own Amazon S3 bucket to hold your export files. Files in this setup are temporary and expire after four hours.  

## CSV exports  
When you export a CSV from the dashboard, Braze emails a download link to the logged-in user. That link points to a ZIP file hosted in Braze's S3 bucket. Inside the ZIP are multiple smaller files that together make up your export.  

> You need to be logged into the Braze dashboard to use the link, and the file will only be available for four hours. After that, the link no longer works and the data is deleted. If you run into repeated failures with very large exports (over 500,000 users), the export may fail. In that case, try splitting your export into smaller groups or fields, or consider setting up a storage partner.  

Reviewer: ```suggestion
You must be logged in to the Braze dashboard to use the link, and the file is available for only four hours. After that, the link no longer works and the data is deleted. If you run into repeated failures with very large exports (over 500,000 users), the export may fail. In that case, try splitting your export into smaller groups or fields, or consider setting up a storage partner.  
```
